index.html

```diff
@@ -31,7 +31,7 @@ <h2 align="center">Todo</h2>
       </li>
     </ol>
            <div class="list-group-item" style="text-align:center;">{{remaining()}} of {{todos.length}} items left</br>
-               <a href="" ng-click="archive()">Clear completed items</a></div>
+               <a href="" ng-click="archive()" style="rotatebutton">Clear completed items</a></div>
```

Comment:
Oh my bad ! I had to add it in the class ... And I did make a mess of it. Sorry for the poor mistake 📦